[Backport release-3_10] Update translators list and fix svg Sardinian Flag and name not show on Options tab

ValPinnaSardinia opened the pull request:
Update Sardinian Translation List

ValPinnaSardinia committed:
Update tsstat.pl

ValPinnaSardinia committed:
Add Sardinian flag 

Add Sardinian Flag for Sardinian Language

agiudiceandrea commented:
Hi @ValPinnaSardinia, probably it is worth incorporating in this PR also this a5bd9cb17db3134a3a9faabe8d203784c79b9853 and this f476ba1fa312e4eff19ea9a9679fcb28a51eebea (in 3,10 qgsoptions.cpp is in src/app instead of src/app/options) changes for 3.10 so they will be merged all together.

ValPinnaSardinia committed:
Update images.qrc

Fix Sardinian flag missing in images.qrc as already done for the other versions (PR #40083)

ValPinnaSardinia committed:
Update qgsoptions.cpp

Add exception for Sardinian language name to workaround upstream issue as already done for other versions,

ValPinnaSardinia commented:
@agiudiceandrea Done. I have added to this PR both changes.
